ee_tests/src/specs/quickstart_deployments.spec.ts

```diff
@@ -131,33 +131,29 @@ describe('Creating new quickstart in OSIO', () => {
     await spacePipelinePage.spaceHeader.deploymentsOption.clickWhenReady();
 
     await browser.sleep(5000);
+
+    support.info('Verifying deployments page');
+
     let spaceDeploymentsPage = new SpaceDeploymentsPage();
```

Comment:
I would call it simply `DeploymentsPage` but the naming conventions should be unified across all page objects.

Reply:
Agreed, SpaceDeploymentsPage is consistent with other pages.